ci-operator/step-registry/gcp/provision/dns-peering-zone/gcp-provision-dns-peering-zone-commands.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+python3 --version 
+export CLOUDSDK_PYTHON=python3
+
+function run_command() {
+    local CMD="$1"
+    echo "Running command: ${CMD}"
+    eval "${CMD}"
+}
+
+CLUSTER_NAME="${NAMESPACE}-${UNIQUE_HASH}"
+GCP_BASE_DOMAIN="$(< ${CLUSTER_PROFILE_DIR}/public_hosted_zone)"
+
+GOOGLE_PROJECT_ID="$(< ${CLUSTER_PROFILE_DIR}/openshift_gcp_project)"
+export GCP_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/gce.json"
+sa_email=$(jq -r .client_email ${GCP_SHARED_CREDENTIALS_FILE})
+if ! gcloud auth list | grep -E "\*\s+${sa_email}"
+then
+  gcloud auth activate-service-account --key-file="${GCP_SHARED_CREDENTIALS_FILE}"
+  gcloud config set project "${GOOGLE_PROJECT_ID}"
+fi
+
+dns_sa=$(yq-go r "${SHARED_DIR}/dns-peering-zone-settings.yaml" serviceAccount)
+target_project=$(yq-go r "${SHARED_DIR}/dns-peering-zone-settings.yaml" targetProject)
+target_network=$(yq-go r "${SHARED_DIR}/dns-peering-zone-settings.yaml" targetNetwork)
+
+echo "$(date -u --rfc-3339=seconds) - INFO: create consumer VPC network..."
+cmd="gcloud compute networks create ${CLUSTER_NAME}-consumer-network --subnet-mode=custom 2>/dev/null"
+run_command "${cmd}"
+cat <<EOF >>"${SHARED_DIR}/dns-peering-zone-settings.yaml"
+consumerNetwork: ${CLUSTER_NAME}-consumer-network
```

Comment:
I guess we can extract the name as a var, so later we can reference it via the var.

Reply:
Do you mean, for example, 
```
consumer_network="${CLUSTER_NAME}-consumer-network"
cmd="gcloud compute networks create ${consumer_network} --subnet-mode=custom 2>/dev/null"
run_command "${cmd}"
cat <<EOF >>"${SHARED_DIR}/dns-peering-zone-settings.yaml"
consumerNetwork: ${consumer_network}

Reply:
right